tests/parametric/test_dynamic_configuration.py

```diff
@@ -298,6 +298,7 @@ def test_telemetry_app_started(self, library_env, test_agent, test_library):
         assert len(events) > 0
 
     @parametrize("library_env", [{**DEFAULT_ENVVARS}])
+    @bug(library="ruby", reason="Telemetry event 'app-client-configuration-change' not found")
```

Comment:
Can you set a JIRA ticket as reason for all `bug` declaration (see https://dd.slack.com/archives/C025TJ4RZ8X/p1726490527322989)